docs/pages/includes/database-access/db-audit-events.mdx

```diff
@@ -0,0 +1,38 @@
+You can view database session activity in the Audit Log
+and after the session is uploaded retrieve with the command `tsh play`.
+
+```code
+# Database session ID will be in a UUID format (ex: 307b49d6-56c7-4d20-8cf0-5bc5348a7101)
+# See the audit log to get a session ID with a key of sid
+$ tsh play --format json <Var name="database.session" />
+```
+
+Example output:
+```json
+    {
+        "cluster_name": "teleport.example.com",
+        "code": "TDB02I",
+        "db_name": "example",
+        "db_origin": "dynamic",
+        "db_protocol": "postgres",
+        "db_query": "select * from sample;",
+        "db_roles": [
+            "access"
+        ],
+        "db_service": "example",
+        "db_type": "rds",
+        "db_uri": "databases-1.us-east-1.rds.amazonaws.com:5432",
+        "db_user": "alice",
+        "ei": 2,
+        "event": "db.session.query",
+        "sid": "307b49d6-56c7-4d20-8cf0-5bc5348a7101",
+        "success": true,
+        "time": "2023-10-06T10:58:32.88Z",
+        "uid": "a649d925-9dac-44cc-bd04-4387c295580f",
+        "user": "alice"
+    }
+```
+
+The Audit Log is viewable in **Activity** under **Management** in the Web UI for users
```

Comment:
```suggestion
The audit log is viewable in **Activity** under **Management** in the Web UI for users
```

Correct me if I'm wrong @ptgott but I don't believe audit log is a proper noun.

Reply:
Yeah, it shouldn't be. The docs are a little inconsistent here, though:

```
 grep -hiRo "audit log" docs/pages | sort | uniq -c
  24 Audit Log
  13 Audit log
 161 audit log
```